prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md

```diff
@@ -0,0 +1,41 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
```

Comment:
It contains all the contents of an HTML document which is Structured content(headings, paragraphs, lists), Embedded content(images, videos, audio, iframes), and forms to collect data from users.